owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/status/RemoteCapability.kt

```diff
@@ -102,3 +104,43 @@ class RemoteCapability {
         filesVersioning = CapabilityBooleanType.UNKNOWN
     }
 }
+
+/**
+ * Enum for Boolean Type in capabilities, with values:
+ * -1 - Unknown
+ * 0 - False
+ * 1 - True
+ */
+enum class CapabilityBooleanType constructor(val value: Int) {
+    UNKNOWN(-1),
+    FALSE(0),
+    TRUE(1);
+
+    val isUnknown: Boolean
+        get() = value == -1
+
+    val isFalse: Boolean
+        get() = value == 0
+
+    val isTrue: Boolean
+        get() = value == 1
+
+    companion object {
+        fun fromValue(value: Int): CapabilityBooleanType? {
+            return when (value) {
+                -1 -> UNKNOWN
+                0 -> FALSE
+                1 -> TRUE
+                else -> null
```

Comment:
Same here, Which is the diference between unknown and null in terms of use? Is it needed to have nullability here?

Reply:
Same but changing `NO_SHARED` for `UNKNOWN`